mitodl/rapid-response-xblock
# Add graph of responses

#### What are the relevant tickets?
Fixes #13 

#### What's this PR do?
Adds D3 and code to create a simple bar chart of responses

#### How should this be manually tested?
 - Load a problem and open it for responses. You should see a blank graph below the button
 - Use this script to populate submissions, one per second: https://gist.github.com/noisecapella/630fbf2c8bb9ea9861c039d564354fae. You will need to replace the `PROBLEM_HASH` value to match the problem you're working with. Note that this will create some new users to match the events submitted.

- While this script is running view the graph. You should see it being updated with new information every three seconds.
 - Run `RapidResponseSubmission.objects.all().delete()` in a Django shell. You should see the graph update to remove all information from it.

#### Screenshots
![screenshot from 2018-03-07 11-57-06](https://user-images.githubusercontent.com/863262/37106025-afcbd59a-21fe-11e8-8812-a7866fbc48dc.png)
